mautic/mautic
# Fix Transport Container Injection

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features"
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #9948... 




#### Description:



#### Steps to test this PR:
To test this PR you need to do this:
1. Activate SendGrid or SES API, you can use Fake API keys since we are not going to send anything.
2. Depending on your selection you need to open on your browser the callback URL, in my case I selected SendGrid so you can open the link `https://domain.com/mailer/sendgrid_api/callback`
3. You will get a 500 error 

After loading this PR:
1. Go to the same callback URL
2. You should see the word `success`  instead of error 500
